src/components/swap/styleds.tsx

```diff
@@ -1,7 +1,7 @@
 import { loadingOpacityMixin } from 'components/Loader/styled'
```

Comment:
Likely modified the wrong file.
Should have created a mod but instead changes were applied to original files